assets/js/elements-utils.js

```diff
@@ -10,3 +10,12 @@ export function removeChildren(node){
         node.removeChild( node.firstChild );
     }
 }
+
+export function makeCopyAllBadgesButtonElement(badgesAsMarkdown) {
+    let div = document.createElement("div");
+    div.innerHTML = `<button className="big" id="all-badges" data-clipboard-text="${badgesAsMarkdown.trim()}">
+        Copy All Badges
+    </button>`;
```

Comment:
you couldn't use a specific function that return html, like as `renderBadge()`?

Reply:
:+1: